src/utils/processOutputInterpreters/trivy-image-output.ts

```diff
@@ -0,0 +1,112 @@
+const LineSplitter = /\r?\n/;
+const dateHeader = /^[-\d]+T[-:.\d]+Z?\s+\033\[\d+m([A-Z]+)\033\[\d+m\s+(.*)$/;
+const borderPattern = /^\+(?:-+\+){6}\s*$/;
+const internalBorderPattern = /^\+(?:(?:-+|\s+)\+){6}\s*$/;
+const rowPattern = /^(?:\|[^|]+){6}\|\s*$/;
+
+const CVEKeys = ['Package', 'VulnerabilityID', 'Severity', 'Title', 'InstalledVersion', 'FixedVersion', 'Description', 'PrimaryURL'];
+const severityRatings: Record<string, number> = {
```

Comment:
This seems like it could be an `enum` in TypeScript.  If we invert the list, we don't need the max rating anymore:
```typescript
enum severityRatings {
  UNKNOWN,
  CRITICAL,
  HIGH,
  MEDIUM,
  LOW,
};

function getRating(key: string) : number {
    return severityRatings[key as keyof typeof severityRatings] || -1;
}
```